src/js/server/apply/router.js

```diff
@@ -275,4 +280,17 @@ function checkHasApplied(req, res, next) {
   }).catch(next);
 }
 
+/**
+ * Intercepts requests to check if applications are still open, redirecting to the dashbaord if not
+ */
+function checkApplicationsOpen(req, res, next) {
+  console.log(process.env.APPLICATIONS_OPEN);
+  if (process.env.APPLICATIONS_OPEN_STATUS === 'closed') {
```

Comment:
Can certainly fix the referring to strings. Do you have any ideas for a cleaner technique then an environment variable?